wallet/wallet.go

```diff
@@ -1144,6 +1167,7 @@ type PurchaseTicketsRequest struct {
 	VotingAddress dcrutil.Address
 	MinConf       int32
 	Expiry        int32
+	SignTx        *bool
```

Comment:
Make this a non-pointer type and flip it to specify if it should remain unsigned (so the default value still results in signing).  It can be named DontSign.  This will simplify the code around line 1130 of createtx.go.